prow/cluster/resources/gatekeeper-constraints/workloads/signifySecretTrustedUsage.yaml

```
@@ -43,7 +43,7 @@ spec:
         command:
           - /tools/entrypoint
         args: []
-        entrypoint_options: '^{.*"args":\["sh","-c","mkdir -p module-chart \\u0026\\u0026 cp -r config\/serverless\/\* module-chart \\u0026\\u0026 \/image-builder --name=serverless-operator --config=\/config\/kaniko-build-config\.yaml --context=\. --dockerfile=Dockerfile"\],"container_name":"test",.*}$'
+        entrypoint_options: '^{.*"args":\["sh","-c","\/image-builder --name=serverless-operator --config=\/config\/kaniko-build-config\.yaml --context=\. --dockerfile=Dockerfile"\],"container_name":"test",.*}$'
```

```suggestion
        entrypoint_options: '^{.*"args":\["sh","-c","\/image-builder --name=serverless-operator --config=\/config\/kaniko-build-config\.yaml --context=\. --dockerfile=components\/operator\/Dockerfile"\],"container_name":"test",.*}$'
```
Or equal to postsubmit use `.+` instead of `Dockerfile` otherwise OPA GAtekeepr will not allow that job to run

---

Thanks, used `.+` as suggested